### ISSUE TITLE
Add support for new Progtest

### ISSUE THREAD

**Issue by Nextross** [keombre/progtest-theme#66]
Progtest got a new look... an ugly one (who would've thought). And the extension is now broken.
![image](https://github.com/user-attachments/assets/5305f45c-a540-4af2-b6f6-d800cf66dcc7)
![image](https://github.com/user-attachments/assets/c6050368-7e4c-4722-9288-e9e09458076b)

**Comment by mstieranka** [comment]
Yeah, that design does look quite ugly... Thanks for the heads up, I'll look into this if I have the time.
